react-native/react/actions/tracker.js

```diff
@@ -84,10 +86,40 @@ export function onFollowHelp (username: string): Action {
   }
 }
 
-export function onCloseFromActionBar (username: string): Action {
-  return {
-    type: Constants.onCloseFromActionBar,
-    payload: {username}
+function trackUser (username: string, state: {tracker: RootTrackerState}): Promise<boolean> {
```

Comment:
I don't think we should be putting promises/functions in stores in general.

I think it make sense that the action that triggers it cares about what happens to it.

for example:
say we block on closing until this returns successful, and if it errors we should dispatch a different action that will show some error info to the user. It would look something like this:

```js
trackUser.then(success => {
  if (!success) throw failure...

  dispatch( closeWindow )
}).catch( () => {
  dispatch( showError )
})
```

In this specific case the result doesn't matter and I'll remove the promise, but that may not always be true